setup.cfg

```diff
@@ -0,0 +1,61 @@
+[metadata]
+name = create-aio-app
+version = attr: create_aio_app.utils.versiontools.get_self_version
+description = The tool that helps quickly create a basis for aiohttp application
+long_description = file: README.md
+long_description_content_type = text/markdown
+platforms = 
```

Comment:
This is usually not needed and it probably means not what you think

Reply:
This information will be presented on pypi. Isn't it so?

Reply:
PyPI lists Trove classifiers

Reply:
We can add a upload to test.pypi, remove this options and see if everything is ok.

Reply:
Maybe we are talking about different parameters, you mean lines 4-7?

![image](https://user-images.githubusercontent.com/11501213/65425554-ec807880-de16-11e9-88a8-0a329ca62f7f.png)